README.md

```diff
@@ -36,6 +36,8 @@ Generate Secrets
 ```bash
 sed -i "s/^SEAFILE_ADMIN_PASSWORD=.*/SEAFILE_ADMIN_PASSWORD=$(pwgen 40 1)/" .env
 sed -i "s/^SEAFILE_MYSQL_ROOT_PASSWORD=.*/SEAFILE_MYSQL_ROOT_PASSWORD=$(pwgen 40 1)/" .env
+sed -i "s/^SEAHUB__SECRET_KEY=.*/SEAHUB__SECRET_KEY=$(pwgen 40 1)/" .env
```

Comment:
Sidenote: Adding the `-s` flag would probably make sense:

>The  pwgen  program generates passwords which are designed to be easily
> memorized by humans, while being as secure  as  possible.   Human-memo‐
> rable  passwords  are  never  going  to be as secure as completely com‐
> pletely random passwords.  In particular, passwords generated by  pwgen
> without  the  -s option should not be used in places where the password
> could be attacked via an off-line brute-force attack.

Reply:
-s does not work on my command line, but -y makes passwords more complicated. Still to avoid issues, I would exclude some chars like of apostrophe.
Therefore my suggestion for discussion: `pwgen -y -r '"`\/,'\' 40 10`